setup/walrus-install.sh

```diff
@@ -94,7 +94,7 @@ curl \
   -o "$tmpdir"/walrus
 
 # Now, let's install this binary.
-install -Dm755 "$tmpdir"/walrus "$install_dir"
+install -m755 "$tmpdir"/walrus "$install_dir"
```

Comment:
```suggestion
mkdir -p "$install_dir"
install -m755 "$tmpdir"/walrus "$install_dir"
```
We'll need to ensure that that directory exists.